src/modal/docs/demo.html

```diff
@@ -6,7 +6,7 @@ <h3 class="modal-title">I'm a modal!</h3>
         <div class="modal-body">
             <ul>
                 <li ng-repeat="item in items">
-                    <a ng-click="selected.item = item">{{ item }}</a>
+                    <a href="#" ng-click="selected.item = item">{{ item }}</a>
```

Comment:
Any reason for this change? I think you should revert this, as I think it would have an adverse effect on the docs page.